src/vaadin-message-avatar.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { AvatarElement } from '@vaadin/vaadin-avatar/src/vaadin-avatar.js';
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+
+registerStyles(
+  'vaadin-message-avatar',
+  css`
+    :host {
+      --vaadin-avatar-outline-width: 0px;
```

Comment:
@Peppe You could use an ignore comment to get rid of Stylelint error:

```css
--vaadin-avatar-outline-width: 0px;
 /* stylelint-disable-line declaration-no-important */
```

I will update our Stylelint config later on.